spec/models/deadline_notification_spec.rb

```diff
@@ -10,98 +10,114 @@
         DeadlineNotification.send_all!
-        dn.reload.should_not be_sent
+        expect(dn.reload).not_to be_sent
       end
     end
   end
 
   describe 'instance method' do
-    before(:each) do
-      @dn = DeadlineNotification.new
-    end
+    let(:notification) { FactoryGirl.create(:deadline_notification) }
+
     describe 'deliver!' do
       it 'calls deliver and update sent' do
-        @dn.should_receive(:deliver)
-        @dn.should_receive(:update_attribute).with(:sent, true)
-        @dn.deliver!
+        expect(notification).to receive(:deliver)
+        expect(notification).to receive(:update_attribute).with(:sent, true)
+        notification.deliver!
       end
     end
+
     describe 'deliver' do
       it 'should do the actual mailer stuff' do
-        mailer = double :mail
-        mailer.should_receive(:deliver)
-        Deliverer.should_receive(:deadline_notification).with(@dn).and_return(mailer)
-        @dn.deliver
+        mailer = instance_double(ActionMailer::MessageDelivery)
+        expect(mailer).to receive(:deliver_now)
+        expect(Deliverer).to receive(:deadline_notification).with(notification).and_return(mailer)
+        notification.deliver
       end
     end
+
     describe 'sendable?' do
+      let(:notification) { DeadlineNotification.new }
+      let(:has_email) { true }
+      let(:due_for_judgement) { true }
+      let(:enabled) { true }
+      let(:withdrawn) { false }
+
       before(:each) do
-        @dn = DeadlineNotification.new
-        @dn.stub(has_email?: true,
-                 due_for_judgement?: true,
-                 enabled?: true,
-                 withdrawn?: false)
-      end
-      it 'is true when has email, is due and is enabled' do
-        @dn.should be_sendable
-      end
-      it 'is false when has no email, is due and is enabled' do
-        @dn.stub(has_email?: false)
-        @dn.should_not be_sendable
-      end
-      it 'is false when has email, is not due and is enabled' do
-        @dn.stub(due_for_judgement?: false)
-        @dn.should_not be_sendable
-      end
-      it 'is false when has no email, is due and is not enabled' do
-        @dn.stub(has_email?: false, enabled?: false)
-        @dn.should_not be_sendable
+        allow(notification).to receive(:has_email?).and_return(has_email)
+        allow(notification).to receive(:due_for_judgement?).and_return(due_for_judgement)
+        allow(notification).to receive(:enabled?).and_return(enabled)
+        allow(notification).to receive(:withdrawn?).and_return(withdrawn)
       end
-      it 'is false when has email, is not due and is not enabled' do
-        @dn.stub(due_for_judgement?: false, enabled?: false)
-        @dn.should_not be_sendable
-      end
-      it 'is false when has no email, is not due and is enabled' do
-        @dn.stub(has_email?: false, due_for_judgement?: false)
-        @dn.should_not be_sendable
+
+      subject { notification.sendable? }
```

Comment:
I like having the subject just under the describe block for that subject so that it becomes obviously correct.